app/src/org/commcare/activities/CommCareHomeActivity.java

```diff
@@ -378,9 +374,9 @@ private void goToFormArchive(boolean incomplete, FormRecord record) {
     void enterRootModule() {
         Intent i;
         if (useGridMenu(org.commcare.suite.model.Menu.ROOT_MENU_ID)) {
-            i = new Intent(getApplicationContext(), MenuGrid.class);
+            i = new Intent(this, MenuGrid.class);
```

Comment:
made these changes when I was debugging the activity navigation issue, but I think this is more correct anyway? Can change back if that's not true